app/src/main/java/app/revanced/all/connectivity/fakewifi/FakeWifiPatch.java

```diff
@@ -0,0 +1,380 @@
+package app.revanced.all.connectivity.fakewifi;
```

Comment:
This class should be in the package `app.revanced.all.connectivity.wifi.fake`. Make sure to apply this in the related patch as well.